lib/swagger/swaggerLoader.ts

```diff
@@ -39,8 +39,12 @@ export class SwaggerLoader implements Loader<SwaggerSpec> {
   }
 
   // TODO reportError
-  public async load(specFilePath: string): Promise<SwaggerSpec> {
-    const swaggerSpec = (await (this.jsonLoader.load(specFilePath) as unknown)) as SwaggerSpec;
+  public async load(specFilePath: string, shouldDeference?: boolean): Promise<SwaggerSpec> {
```

Comment:
I don't find `deference` has the meaning of resolve reference. Can we use resolveRef instead?

Reply:
no, as discussed in scrum, resolveRef will remove all other attributes such as "readOnly", "x-ms-secret" when resolving the references.